PayPal/src/test/java/com/braintreepayments/api/paypal/PayPalVaultRequestUnitTest.java

```diff
@@ -126,4 +128,22 @@ public void createRequestBody_sets_userAuthenticationEmail_when_not_null() throw
 
         assertTrue(requestBody.contains("\"payer_email\":" + "\"" + payerEmail + "\""));
     }
+
+    @Test
+    public void createRequestBody_sets_enablePayPalSwitch() throws JSONException {
+        int versionSDK = Build.VERSION.SDK_INT;
+        PayPalVaultRequest request = new PayPalVaultRequest(true);
+
+        request.setEnablePayPalAppSwitch(true);
+        String requestBody = request.createRequestBody(
+                mock(Configuration.class),
+                mock(Authorization.class),
+                "success_url",
+                "cancel_url"
+        );
+
+        assertTrue(requestBody.contains("\"launch_paypal_app\":true"));
+        assertTrue(requestBody.contains("\"os_type\":" + "\"Android\""));
+        assertTrue(requestBody.contains("\"os_version\":" + versionSDK));
```

Comment:
Should we also check for MERCHANT_APP_RETURN_URL_KEY?

Reply:
For now, I didn't consider it necessary. You're absolutely right, it's something that should be tested. For the moment, I updated the TODO to remind myself to add this assert later.